### PR TITLE
chore: update README for EAS device builds, fix EAS config

### PR DESCRIPTION
**Pull request by robinheinze**
# Description

[Trello Card](58-docs-document-flow-for-releasing-beta-builds-for-testers)

Summary of the changes and any helpful notes for reviewers and testers. 

- Fixes EAS profiles for device builds
- Adds sections to the README for building to physical devices, namely iOS devices which are more difficult. 

# Checklist:

- [x] I have done a thorough self-review of my code
- [ ] I have tested with a screen reader and font-scaling turned on and added necessary accessibility features 
- [ ] I have run tests and linter

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,20 +16,59 @@ Note: The steps below will have to be run when either of the following happen:
...
+Great for developing on device for testing features like screenreader support
+
+Android
+
+1. Follow emulator directions above
+2. Instead of dragging build to emulator, use `adb install` to install on connected device
+
+iOS
+
+1. Run `yarn build:dev:device`
```
Comment (frankcalise) [inline README.md:52]: I would add in that you can optionally pass `--local` to not wait on the EAS servers if you want to build locally

File: README.md [hunk 2]
```diff
@@ -16,20 +16,59 @@ Note: The steps below will have to be run when either of the following happen:
...
+5. Install via QR Code
+6. Run `yarn start` in your terminal.
+7. Scan the QR code that comes up in the terminal output with your device. This should connect your phone app installation to the metro server running on your computer.
+8. Good to go!
+
+#### Internal Distribution
+
+This is for distributing the app to internal testers to run on their devices. The app is in release mode, so this isn't ideal for active development
+
+1. Run `yarn build:preview:device`
```
Comment (frankcalise) [inline README.md:65]: Also can be done locally I think

**Review by frankcalise — APPROVED** [review]
Looks good! Added optional notes about running builds locally

File: README.md (hunk 1 above)
Comment (robinheinze) [reply README.md:52]: Does that work for physical iOS devices? I've never installed a plain .ipa file to a device without XCode

Comment (robinheinze) [reply README.md:52]: Updated to include a local option!

**Pull request merged**